clear-containers-image/_service-template

```diff
@@ -0,0 +1,13 @@
+<services>
+ <service name="tar_scm">
+    <param name="scm">git</param>
+    <param name="url">https://github.com/erick0z/clear-containers-image.git</param>
```

Comment:
We need to use a repository under clearcontainers group:

@jodh-intel @sameo thoughts?

Reply:
Agreed - it's not a good idea to download from a users repo.

But, rather than copying files from https://download.clearlinux.org/releases/ to a github repo, can we instead use one of these plugins to just download the image file from https://download.clearlinux.org using one of the other OBS services such as `download_url`?:

- https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#All_OBS_services_available

Reply:
If that doesn't work, we could look at using the `download_files` service which sounds like it might be able to grab the spec file from this repo and download the associated `clear-containers.img` file?:

- https://github.com/clearlinux-pkgs/clear-containers-image

Reply:
@jodh-intel we already use those OBS services (downloading golang tarball, for instance). The thing is, in this case, the format of the file. Since the image is the ``source`` of the package, for debian builds, it must be a tarball. That's why I used my personal repo for testing. Anyway, I think a tarballed image will be published in clearlinux.org, making it easier for us.